.github/workflows/ci.yml

```diff
@@ -20,6 +20,10 @@ jobs:
         with:
           java-version: ${{ matrix.jdk }}
 
+      - name: Check sbt formatting
```

Comment:
I separated this because we don't need to run it for every Scala version. I don't have a strong preference either way, so I joined both checks in 130eb56.

Reply:
The same is true for `scalafmtCheckAll` - we expect formatted code to be the same regardless of the Scala version used, and if by some weird reason scalafmt formats it differently we have to abide by one specific version in our codebase. Feel free to re-add the Scala version GitHub Actions condition again :)

Reply:
I'm assuming we run `scalafmtCheckAll` for all versions not because we're expecting differences in the format, but because we have different files compiled for different versions.

Reply:
Makes sense, I forgot about that 👍